### PR TITLE
ZCS-13419 : Install license-daemon-service with zimbra setup

### PR DESCRIPTION
**Pull request by umagmrit**

### CODE REVIEW

File: rpmconf/Install/zmsetup.pl [hunk 1]
```diff
@@ -5787,6 +5789,27 @@ sub configCreateCert {
     }
   }
 
+  if (isInstalled("zimbra-license-daemon")) {
+	  if ( !-f "/opt/zimbra/ssl/zimbra/server/server.crt") {
+		  progress ( "Creating SSL certificate..." );
+		  $rc = runAsZimbra("/opt/zimbra/bin/zmcertmgr createcrt $needNewCert");
```
Comment (dasiyogesh) [inline rpmconf/Install/zmsetup.pl:5795]: can we move 
`		  $rc = runAsZimbra("/opt/zimbra/bin/zmcertmgr createcrt $needNewCert");
		  if ($rc != 0) {
			  progress ( "failed.\n" );
			  exit 1;
		  } else {
			  progress ( "done.\n" );
		  }
` 
out of if-else block as thats getting repeated

Comment (umagmrit) [reply rpmconf/Install/zmsetup.pl:5795]: there may be a case if it not goes into the IF...ELSIF block

**Review by dasiyogesh — APPROVED** [review]

**Review by GaneshAnarse — APPROVED** [review]

File: rpmconf/Install/zmsetup.pl [hunk 2]
```diff
@@ -7225,6 +7262,8 @@ sub applyConfig {
 
   configInitMta();
 
+  configureLicenseDaemonService();
```
Comment (silentsakky) [inline rpmconf/Install/zmsetup.pl:7265]: inside this we are checking if service is enabled or not but service enablement status is set in ldap in configSetEnabledServices method in next line

Comment (umagmrit) [reply rpmconf/Install/zmsetup.pl:7265]: before configSetEnabledServices, all installed packages will be enabled. https://github.com/Zimbra/zm-build/blob/b1623ed42a66131fc2540363242cd9926eb143e0/rpmconf/Install/zmsetup.pl#LL668C33-L668C55

**Review by silentsakky — APPROVED** [review]

**Pull request merged**